django/test/client.py

```diff
@@ -260,7 +262,8 @@ class RequestFactory:
     Once you have a request object you can pass it to any view function,
     just as if that view had been hooked up using a URLconf.
     """
-    def __init__(self, **defaults):
+    def __init__(self, json_encoder=DjangoJSONEncoder, **defaults):
```

Comment:
What about using keyword-only args?
```python
def __init__(self, *, json_encoder=DjangoJSONEncoder, **defaults):
```

Reply:
Is it worth it?

Reply:
TBH I'm already doing all the doc changes and I'm pro this change so happy to do it.

Reply:
I think it is better because it is more explicit and future proof if you want to add other params.
But wait for other comments :)